### PR TITLE
Create UI

### PR DESCRIPTION
**Pull request by ice949**
### New Features ###
- Created a new Calculator.js file.
- Created a React component that matches this design
- Use class-based components

### CODE REVIEW

**Review by zhadier — CHANGES_REQUESTED** [review]
Made this review by mistake, just ignore this :)

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import Calculator from './components/Calculator';
 import './index.css';
 
 ReactDOM.render(
-  <App />,
+  <Calculator />,
```
Comment (zhadier) [inline src/index.js]: - [ ] Great job on the overall implementation. But one thing you must do is to keep the `App.js` file in the project and then render calculator inside the App.js. In all react applications, the App.js is the default entry point for the application and it is where the routes are defined (You'll learn about routing soon). So it is imperative that you keep the App.js file to be able to do the future milestones aswell. Addiitonally, please make sure the App.js is a class based component to conform to the project requirement.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ice949  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ GitFlow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (ice949) [reply src/index.js]: @zhadier So what of the index.js should I also keep it

**Review by julie-ify — APPROVED** [review]
Hi @ice949,  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**